src/main/java/core/basesyntax/Factorial.java

```diff
@@ -6,6 +6,11 @@
  */
 public class Factorial {
     public int getFactorial(int number) {
-        return 0;
+        int result = 1;
+        if (number == 1 || number == 0) {
+            return result;
+        }
+        result = number * getFactorial(number - 1);
```

Comment:
remove local variable, it's redundant in this case

Reply:
Done